src/libs/OptionsListUtils.ts

```diff
@@ -1654,6 +1711,36 @@ function filterReports(reports: ReportUtils.OptionData[], searchTerms: string[])
     return filteredReports;
 }
 
+function filterWorkspaceChats(reports: ReportUtils.OptionData[], searchTerms: string[]): ReportUtils.OptionData[] {
```

Comment:
I think we can simplify this function by filter by text the same [condition](https://github.com/callstack-internal/Expensify-App/blob/main/src/pages/WorkspaceSwitcherPage/index.tsx#L126) at `Filter by workspace` page, since the WS option doen't include `isThread`, `login` and `subtitle` are always set to workspace.

https://github.com/user-attachments/assets/eafa775c-7104-4a23-9913-ccce9c1d6ebf

Reply:
Updated